### PR TITLE
Place the 3rd party JavaScript and CSS files under the third_party directory

### PR DESCRIPTION
**Pull request by LeZhang2016**
fix issue #9 and #18

### CODE REVIEW

**Comment by LeZhang2016** [comment]
@jwhui, could we just put the necessary Java script and CSS files (`angular.min.js`, `angular-animate.min.js`, `angular-aria.min.js`,` angular-material.min.js`,` angular-material.min.css`) into the repo as the third party lib, not all files?  The Size of all files is more than 15MB.

**Comment by jwhui** [comment]
I think it is fine to include a subset of files, as long as the `README.md` clearly lists that this only includes a subset of the files.

File: third_party/angular-material/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# AngularJS
+
+## URL
+
+https://github.com/angular/angular.js
```
Comment (jwhui) [inline third_party/angular-material/README.md]: Prefer a link to the actual release or git hash.

**Comment by bukepo** [comment]
@LeZhang2016 Could you please also update the `script/bootstrap` and removing nodejs and bower installation? Thanks.

**Comment by LeZhang2016** [comment]
@jwhui, @bukepo , I have updated this PR, could you help me review this PR, and let me know if you have any other comments or suggestions for the PR. Thanks a lot for your help!

File: third_party/angular-material/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+MIT License
+
+## License File
+
+[LICENSE](repo/LICENSE)
+
+## Description
+Material Design is a specification for a unified system of visual, motion, and interaction design that adapts across different devices. Our goal is to deliver a lean, lightweight set of AngularJS-native UI elements that implement the material design specification for use in AngularJS single-page applications (SPAs).
+
+This repo only includes the minify version files and LICENSE file.
```
Comment (jwhui) [inline third_party/angular-material/README.md]: Change to: "Note: only the minimum set of files necessary to support the OpenThread project are included here."

File: third_party/angular-material/README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# AngularJS
```
Comment (jwhui) [inline third_party/angular-material/README.md]: Change to: "Material Design for AngularJS"

File: third_party/angular/README.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+MIT License
+
+## License File
+
+[LICENSE](repo/angular/LICENSE.md)
+
+## Description
+Angular is a development platform for building mobile and desktop web applications using Typescript/JavaScript (JS) and other languages.
+
+This repo only includes the minified version files and LICENSE file.
```
Comment (jwhui) [inline third_party/angular/README.md]: Change to: "Note: only the minimum set of files necessary to support the OpenThread project are included here."

File: third_party/angular/README.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
+# Angular
```
Comment (jwhui) [inline third_party/angular/README.md]: Change to: "AngularJS"

**Comment by LeZhang2016** [comment]
@jwhui, @lanyuwen, looking forward to your comments. Thanks.

**Comment by lanyuwen** [comment]
Agree that it's fine to only include a subset of files, however, we should organize these files following the original directory structures rather than put all files flat, or it would make it hard to maintain moving forward.

**Comment by LeZhang2016** [comment]
@lanyuwen , @jwhui , @bukepo, Thanks a lot for your suggestions. I think i provided incorrect URLs about the `angularJS` and `angular-material` in `README` file. The previous URLs link to the source repos of the two frameworks, in which users can not obtain the built JavaScript and CSS files directly. The new URLs are from the official website, the built JS and CSS can be obtained from these URLs directly now.
AngularJS : https://code.angularjs.org/1.6.4/
AngularJS Material : https://github.com/angular/bower-material

**Pull request merged**